backend/app/routers/routers.py

```diff
@@ -16,92 +18,99 @@
 @router.post("/login")
 async def login(
     form_data: Annotated[OAuth2PasswordRequestForm, Depends()], request: Request
-) -> Token:
+) -> AuthToken:
+    supabase_auth_service = SupabaseAuthService(settings=settings)
```

Comment:
Should use DI instead?

Reply:
Should inject AuthService, not SupabaseAuthService -- the router doesn't need to know which implementation was chosen by DI.

Reply:
AuthService has been added to DI.

Reply:
Along with UserService and Settings.